flake8rc

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E125,D100,D101,D102,D200,P101,FI15,FI16,FI12,FI11,FI17,FI50,FI53,FI54
+max-line-length = 120
```

Comment:
This can be moved to setup.cfg? I'd like overlap between prospector and this here if possible.

Reply:
Yes, http://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations